src/librustdoc/html/render/mod.rs

```diff
@@ -453,6 +453,7 @@ fn settings(root_path: &str, suffix: &str, themes: &[StylePath]) -> Result<Strin
         ("auto-hide-method-docs", "Auto-hide item methods' documentation", false).into(),
         ("auto-hide-trait-implementations", "Auto-hide trait implementation documentation", false)
             .into(),
+        ("auto-collapse-implementors", "Auto-hide implementors of a trait", true).into(),
```

Comment:
I don't think we should reintroduce this

Reply:
It's available for all other impl blocks, so should be weird to not be there, no?

Reply:
No? The other settings are different. We removed this because we stopped showing methods, which is what made the impl section so big. The impl section is no longer that big.

Reply:
Depends on how many associated items you have?

Reply:
It's super unlikely to have that many. I don't want to add a setting to deal with an extremely rare edge case, and I don't think a regression fix is the place to do so _anyway_. Overall we've learned that these settings are confusing and we've been moving towards handing things sensibly in ways that don't need settings (e.g. the way we choose to collapse associated items _when there are tons of them_ so that the settings for those can be collapsed)

Reply:
Fine by me.